src/components/MyKiva/BadgeInProgress.vue

```diff
@@ -27,6 +27,7 @@
 			slides-to-scroll="visible"
 			:slide-max-width="singleSlideWidth"
 			@interact-carousel="onInteractCarousel"
+			@change="onInteractCarousel"
```

Comment:
With `@change` and `@interact-carousel` both calling the same method, do we get double analytics events?

Reply:
What was the issue with `@interact-carousel`?